src/theme/base/colors.ts

```diff
@@ -2,6 +2,8 @@ const colors = {
   mainBackground: '#F0F2F3',
   cardPrimaryBackground: '#0A906E',
   textColor: '#0B0B0B',
+  white: '#ffffff',
+  cardBackground: '#ffffff',
```

Comment:
why we need to separate the name, while they are the same color.

Reply:
Yes. I removed it